Samples/WebFormsSample/BasePage.cs

```diff
@@ -0,0 +1,10 @@
+﻿// MIT License.
+
+using System.Web.UI;
```

Comment:
no need to have a sample and a test with this. the test is probably a better location so we detect regressions

Reply:
It wasn't clear for me what should I do with this? move it to another separate test?
The idea of this PR at this moment is to replicate the issue and hopefully get some help on what is the solution to fix it?  

I'm currently blocked on this.

Reply:
Gotcha - I'll clone it later today and take a look

Reply:
Let me know if that worked for you.